packages/glob/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@actions/glob",
+  "version": "1.0.0",
```

Comment:
probably start with `0.1`?

Reply:
Should this be in core?

Reply:
I think size would be the only consideration - core folks (everyone) that doesn't need glob.  The other way around isn't a scenario IMHO (need glob but not core)

Reply:
Open to whatever, here's the factors i considered:
- A separate module allows independent versioning. E.g. can start out with `0.1-preview`
- It also aligns with npm philosophy, small targeted packages.
- Size when installing minimatch on a fresh directory: 80kb uncompressed, 20kb compressed 

Would this align more with core, or with io?